### DISCUSSION TITLE
Google auth

### DISCUSSION THREAD

**Discussion by JulioSM98** [Leantime/leantime#509] [Ideas]
Permitir login con google api

**Comment by marcelfolaron** [comment]
Currently working on ldap integration and will look for 3rd party auth after that.

**Reply by manuelalvarez** [reply]
Love Leantime! Kudos to all working on it. Any updates on the ldap integration?

**Comment by ferdimorais** [comment]
Hello, any news on this thread? Loving Leantime, and it would be extremely good if there was Google Auth. Willing to help with it!

**Comment by marcelfolaron** [comment]
Ldap is done, google auth is not and we could certainly use help with that.
Happy to walk you through the code for authentication. I just cleaned
things up that should make it easier to integrate.
